getting-started/data-node-authentication.md

```diff
@@ -185,9 +185,264 @@ timescaledb.passfile = 'pgpass.txt'
+Signing Requests (CSRs), and the certificate of the certificate
+authority is what is used as a root certificate for other parties.
+
+A common way to set up certificate authentication when demonstrating
+it is using self-signed certificates for the clients, but in this
+chapter you will instead use a toy certificate authority. This is more
+similar to how it will work in a real production setup.
+
+>:WARNING: The Certificate Authority key is central to your security
+>and needs to be treated with care. Ideally, the key should be
+>generated and used on a secured machine dedicated for this
+>purpose. Typically, the key should not leave the machine and you
+>should even consider the risk of backups being compromised.
+
+To generate a key into the file `auth.key` you can use `openssl`:
+
+```bash
+openssl genpkey -algorithm rsa -out auth.key
+```
+
+To generate a root certificate for the toy CA, you can generate a
+self-signed root certificate using `openssl`.
+
+```
+$ openssl req -new -key auth.key -days 3650 -out root.crt -x509
+You are about to be asked to enter information that will be incorporated
+into your certificate request.
+What you are about to enter is what is called a Distinguished Name or a DN.
+There are quite a few fields but you can leave some blank
+For some fields there will be a default value,
+If you enter '.', the field will be left blank.
+-----
+Country Name (2 letter code) [AU]:US
+State or Province Name (full name) [Some-State]:New York
+Locality Name (eg, city) []:New York
+Organization Name (eg, company) [Internet Widgits Pty Ltd]:Example Company Pty Ltd
+Organizational Unit Name (eg, section) []:
+Common Name (e.g. server FQDN or YOUR name) []:http://cert.example.com/
+Email Address []: 
+```
+
+The common name can be anything for this certificate, but here the
+common name contains the (imaginary) location of the root certificate,
+which in theory would allow a user to download the certificate from a
+known location and verify that it is the correct one.
+
+Remember that the root certificate only contains public information and
+can be distributed freely between machines. Actually, it is more secure
+to distribute it widely since anybody interested in using the root
+certificate can be more certain that she got the right certificate if it
+is available at many locations.
+
+### Generating keys and certificates for the instances
+
+Once you have a root certificate authority you can create certificates
+for the both the data node and the access node. On the data node you
+need a signed certificate to certify to the access node that the data
+node is valid. On the access node you need to have a signed
+certificate so that the access node can sign user certificates.
+
+The default names for the instance key and certificate is `server.key`
+and `server.crt` respectively and they are placed in the data
+directory of the instance, so let's use those names. If you do not
```

Comment:
I remember we had a PR which purpose was to place those files in a user/certificate directory inside the data directory. I had an impression that we merged it or we just not expose those details here?